src/medicine/main.py

```diff
@@ -1,47 +1,57 @@
-import pickle
-from icecream import ic
-from medicine.medicine_classes import Medicine
+import json
+from typing import List
```

Comment:
Co do wrzucania nowych bibliotek itd. to nie wiem czy na razie będę się w to bawił. W sensie wiem jak się używa `JSON`a, ale niektórych fragmentów kodu nie rozumiem, a teraz to ja się assemblerem bawię bardziej (hlip hlip). Ale zobaczę

Reply:
jasne. Od tego są PR żeby zadawać pytania itp.

Reply:
A da się tylko część pull requesta zaakceptować? czy all or nothing?

Reply:
nie, wszystko trzeba. Ale od tego są komenatrze żeby powiedzieć że to chcesz a tamto nie. Wtedy osoba która proponuje zmiany musi poprawić